### ISSUE TITLE
Empty release PR on `release-please--branches--master` is created

### ISSUE THREAD

**Issue by alojzy231** [google-github-actions/release-please-action#805]
### TL;DR

`release-please-action` creates release PR on `release-please--branches--master` branch, with an empty commit, with no description content.

### Expected behavior

`release-please--branches--master` branch is not created, nor the release PR. 
All changes are included in separate, individual release PRs so there is no need to create any additional PR.

### Observed behavior

When using `separate-pull-requests`, `monorepo-tags` and `node-workspace` plugin in config:
```json
{
  "bootstrap-sha": "4c0826e409e70c0e1babf60fe1266c7830edc600",
  "release-type": "node",
  "separate-pull-requests": true,
  "plugins": ["node-workspace"],
  "monorepo-tags": true,
  "pull-request-title-pattern": "chore(${component}): release ${version}",
  "packages": {...}
}
```
`release-please-action` creates release PR on `release-please--branches--master` branch, with an empty commit, with no description content.

### Action YAML

```yaml
on:
  push:
    branches:
      - master

permissions:
  contents: write
  pull-requests: write

name: release-please

jobs:
  release-please:
    runs-on: ubuntu-latest
    steps:
      - uses: google-github-actions/release-please-action@v3
        with:
          release-type: node
          package-name: release-please-action
          command: manifest
          monorepo-tags: true
          token: ${{ secrets. PERSONAL_ACCESS_TOKEN }}
```

### Log output

```text
Run google-github-actions/release-please-action@v3
❯ Fetching release-please-config.json from branch master
❯ Fetching .release-please-manifest.json from branch master
✔ Building releases
✔ Building strategies by path
❯ apps/docs: node
❯ apps/web: node
❯ packages/ui: node
❯ Fetching release-please-config.json from branch master
❯ Fetching .release-please-manifest.json from branch master
✔ Building pull requests
✔ Building strategies by path
❯ apps/docs: node
❯ apps/web: node
❯ packages/ui: node
❯ Fetching apps/docs/package.json from branch master
❯ Fetching apps/web/package.json from branch master
❯ Fetching packages/ui/package.json from branch master
✔ Collecting release commit SHAs
❯ release search depth: 400
❯ Fetching releases with cursor undefined
❯ Found release for path apps/web, web-v1.3.1
❯ Found release for path apps/docs, docs-panel-v1.4.0
❯ Found release for path packages/ui, ui-master-v1.4.0
❯ release for path: apps/web, version: 1.3.1, sha: d83ff80dc18776af3e33b7aae849cacd3d22ac07
❯ release for path: apps/docs, version: 1.4.0, sha: 3e1107a48913ed778e84b9126daa125989a02044
❯ release for path: packages/ui, version: 1.4.0, sha: 4b7c91416803ff150b3bbe8c97e318148dd969ad
✔ Collecting commits since all latest releases
❯ commit search depth: 500
❯ Set(3) {
  'd83ff80dc18776af3e33b7aae849cacd3d22ac07',
  '3e1107a48913ed778e84b9126daa125989a02044',
  '4b7c91416803ff150b3bbe8c97e318148dd969ad'
}
❯ Fetching merge commits on branch master with cursor: undefined
❯ Backfilling file list for commit: 5a69e3f6bac44ad5551b574d879bbdfbe66aca73
❯ Found 1 files
❯ Backfilling file list for commit: 4ffc70f60cd6d5dd90bc18fb8172f3ea06530aea
❯ Found 1 files
❯ Backfilling file list for commit: ec30c348c5a8d584fff2b997fe1c536cf4b8984a
❯ Found 1 files
❯ Backfilling file list for commit: 2ba67e5ae80f246e77ea41d916af9494cd4bd5d1
❯ Found 1 files
❯ Backfilling file list for commit: b35772e495e85a8ea6c97c4adfa2b4afb44d6229
❯ Found 1 files
❯ Backfilling file list for commit: e6cdb39615fe99b01a3aee37771387e38cf49ad8
❯ Found 1 files
❯ Backfilling file list for commit: b1b0609202c09615a6c8736027729dd00b54946d
❯ Found 1 files
❯ Backfilling file list for commit: e2787a772fe768c3460b5755afe132296602dcf4
❯ Found 1 files
❯ Backfilling file list for commit: 393aaacdd50ebbdfca0551bcf05e5c53c68f76d9
❯ Found 1 files
❯ Backfilling file list for commit: c4f05db9e687dc73c89defdd7a96f056f82fea9e
❯ Found 1 files
❯ Backfilling file list for commit: 9ff1c321b03c6cf4cf71deed57dbe683d4a5002b
❯ Found 1 files
❯ Backfilling file list for commit: 6016ebe8042354b49e713b2288ba8142470e9130
❯ Found 2 files
❯ Backfilling file list for commit: 281d60da443f60039b827bd89518ea8650816ec0
❯ Found 2 files
❯ Backfilling file list for commit: e29322524be50d857ebf29b3e08f244020e0e0e4
❯ Found 1 files
❯ Backfilling file list for commit: 3074eec5c089456e4eec0cb11dac98a5a88c2df3
❯ Found 1 files
❯ Backfilling file list for commit: 69b00b167ba05873043edc39c565114de037c564
❯ Found 1 files
❯ Backfilling file list for commit: 044077faddd8d90eb558dda0a2755cbf0466805e
❯ Found 2 files
❯ Backfilling file list for commit: 3d9f9ec95f81b2ff7041e16485131bc84453de3d
❯ Found 1 files
✔ Splitting 6 commits by path
✔ Building candidate release pull request for path: apps/docs
❯ type: node
❯ targetBranch: master
❯ commits: 2
✔ Considering: 2 commits
❯ component: docs-panel
❯ pull request title pattern: chore(${component}): release ${version}
⚠ pullRequestTitlePattern miss the part of '${scope}'
⚠ pullRequestTitlePattern miss the part of '${scope}'
✔ Building candidate release pull request for path: apps/web
❯ type: node
❯ targetBranch: master
❯ commits: 2
✔ Considering: 2 commits
❯ component: web
❯ pull request title pattern: chore(${component}): release ${version}
✔ Building candidate release pull request for path: packages/ui
❯ type: node
❯ targetBranch: master
⚠ pullRequestTitlePattern miss the part of '${scope}'
❯ commits: 2
✔ Considering: 2 commits
❯ component: ui-master
❯ pull request title pattern: chore(${component}): release ${version}
❯ running plugin: NodeWorkspace
✔ Running workspace plugin
✔ Found 3 in-scope releases
✔ Building list of all packages
❯ Found candidate pull request for path: apps/docs
❯ Fetching apps/docs/package.json from branch master
❯ Found candidate pull request for path: apps/web
❯ Fetching apps/web/package.json from branch master
❯ Found candidate pull request for path: packages/ui
❯ Fetching packages/ui/package.json from branch master
✔ Building dependency graph for 3 packages
✔ building graph order, existing package names: docs-panel,web,@workspace/ui-master
❯ visiting docs-panel, path: 
❯ marking docs-panel as visited and adding docs-panel to order
❯ visiting web, path: 
❯ marking web as visited and adding web to order
❯ visiting @workspace/ui-master, path: 
✔ visiting docs-panel next
❯ visiting docs-panel, path: @workspace/ui-master
❯ [object Object] already visited
✔ visiting web next
❯ visiting web, path: @workspace/ui-master
❯ [object Object] already visited
❯ marking @workspace/ui-master as visited and adding @workspace/ui-master to order
✔ Updating 3 packages
❯ package: @workspace/ui-master
❯ version: 1.4.1 from release-please
❯ package: docs-panel
❯ version: 1.4.1 from release-please
❯ package: web
❯ version: 1.4.0 from release-please
✔ Updating exising candidate pull request for @workspace/ui-master, path: packages/ui
✔ Updating @workspace/ui-master to 1.4.1
✔ Updating exising candidate pull request for docs-panel, path: apps/docs
✔ Updating docs-panel to 1.4.1
✔ docs-panel.@workspace/ui-master updated to 1.4.1
✔ appending dependency notes to changelog
✔ appending dependency notes to changelog
✔ Updating exising candidate pull request for web, path: apps/web
✔ Updating web to 1.4.0
✔ web.@workspace/ui-master updated to 1.4.1
✔ appending dependency notes to changelog
✔ appending dependency notes to changelog
✔ Merging 3 in-scope candidates
✔ Merging 3 pull requests
⚠ pullRequestTitlePattern miss the part of '${scope}'
⚠ pullRequestTitlePattern miss the part of '${component}'
⚠ pullRequestTitlePattern miss the part of '${version}'
✔ Post-processing 4 in-scope candidates
✔ Looking for open release pull requests
✔ found 0 open release pull requests.
✔ Looking for snoozed release pull requests
✔ found 0 snoozed release pull requests.
❯ Fetching .release-please-manifest.json from branch master
❯ Fetching packages/ui/package-lock.json from branch master
❯ Fetching apps/docs/package-lock.json from branch master
❯ Fetching apps/web/package-lock.json from branch master
✔ Starting GitHub PR workflow...
⚠ file packages/ui/package-lock.json did not exist
⚠ file apps/docs/package-lock.json did not exist
❯ Fetching packages/ui/npm-shrinkwrap.json from branch master
⚠ file apps/web/package-lock.json did not exist
❯ Fetching apps/docs/npm-shrinkwrap.json from branch master
⚠ file packages/ui/npm-shrinkwrap.json did not exist
❯ Fetching apps/web/npm-shrinkwrap.json from branch master
⚠ file apps/docs/npm-shrinkwrap.json did not exist
❯ Fetching packages/ui/samples/package.json from branch master
⚠ file apps/web/npm-shrinkwrap.json did not exist
❯ Fetching apps/docs/samples/package.json from branch master
⚠ file packages/ui/samples/package.json did not exist
❯ Fetching apps/web/samples/package.json from branch master
⚠ file apps/docs/samples/package.json did not exist
❯ Fetching packages/ui/CHANGELOG.md from branch master
⚠ file apps/web/samples/package.json did not exist
❯ Fetching apps/docs/CHANGELOG.md from branch master
❯ Fetching apps/web/CHANGELOG.md from branch master
❯ Fetching apps/web/package.json from branch master
⚠ file changelog.json did not exist
❯ Fetching changelog.json from branch master
❯ Fetching .release-please-manifest.json from branch master
✔ Starting GitHub PR workflow...
✔ Successfully found branch HEAD sha "5a69e3f6bac44ad5551b574d879bbdfbe66aca73".
❯ Fetching apps/docs/package.json from branch master
❯ Fetching changelog.json from branch master
⚠ file changelog.json did not exist
❯ Fetching .release-please-manifest.json from branch master
✔ Starting GitHub PR workflow...
✔ Successfully found branch HEAD sha "5a69e3f6bac44ad5551b574d879bbdfbe66aca73".
✔ Skipping branch creation step...
✔ Skipping branch creation step...
✔ Got the latest commit tree
✔ Got the latest commit tree
❯ Fetching packages/ui/package.json from branch master
❯ Fetching changelog.json from branch master
⚠ file changelog.json did not exist
❯ Fetching .release-please-manifest.json from branch master
✔ Starting GitHub PR workflow...
✔ Successfully created a tree with the desired changes with SHA 673eb9ac7a667b41c819e2130fa333b2f23ddd2b
✔ Successfully created a tree with the desired changes with SHA 703c710601822d1e479d072d6c574e993b582e85
✔ Successfully found branch HEAD sha "5a69e3f6bac44ad5551b574d879bbdfbe66aca73".
✔ Skipping branch creation step...
✔ Successfully created commit. See commit at https://api.github.com/repos/alojzy231/release-please-playground/git/commits/dd1c824c1a3beeb43906a56659e2a7c3fc72edf2
✔ Updating reference heads/release-please--branches--master to dd1c824c1a3beeb43906a56659e2a7c3fc72edf2
✔ Successfully created commit. See commit at https://api.github.com/repos/alojzy231/release-please-playground/git/commits/d47e6998a3db79f1774cb41dbff767d85f833563
✔ Updating reference heads/release-please--branches--master--components--docs-panel to d47e6998a3db79f1774cb41dbff767d85f833563
✔ Got the latest commit tree
✔ Successfully updated reference release-please--branches--master to dd1c824c1a3beeb43906a56659e2a7c3fc72edf2
✔ Successfully created a tree with the desired changes with SHA b267afcf648d6616a933932d5253130bda4cf3f3
✔ Successfully updated reference release-please--branches--master--components--docs-panel to d47e6998a3db79f1774cb41dbff767d85f833563
✔ Successfully created commit. See commit at https://api.github.com/repos/alojzy231/release-please-playground/git/commits/20338541b49ac24438e8b4db8536290d91334f22
✔ Updating reference heads/release-please--branches--master--components--ui-master to 20338541b49ac24438e8b4db8536290d91334f22
✔ Successfully found branch HEAD sha "5a69e3f6bac44ad5551b574d879bbdfbe66aca73".
✔ Successfully updated reference release-please--branches--master--components--ui-master to 20338541b49ac24438e8b4db8536290d91334f22
✔ Skipping branch creation step...
✔ Got the latest commit tree
✔ Successfully created a tree with the desired changes with SHA ad7b4f10489b028f31b54d80649858feb21d39c5
✔ Successfully created commit. See commit at https://api.github.com/repos/alojzy231/release-please-playground/git/commits/ab9193c7fe10a7c43046a91c79a20a9c9c8d201d
✔ Updating reference heads/release-please--branches--master--components--web to ab9193c7fe10a7c43046a91c79a20a9c9c8d201d
✔ Successfully opened pull request available at url: https://api.github.com/repos/alojzy231/release-please-playground/pulls/17.
✔ Successfully opened pull request: 17.
✔ Successfully updated reference release-please--branches--master--components--web to ab9193c7fe10a7c43046a91c79a20a9c9c8d201d
✔ Successfully opened pull request available at url: https://api.github.com/repos/alojzy231/release-please-playground/pulls/18.
✔ Successfully opened pull request: 18.
✔ Successfully added labels autorelease: pending to issue: 17
✔ Successfully opened pull request available at url: https://api.github.com/repos/alojzy231/release-please-playground/pulls/19.
✔ Successfully opened pull request: 19.
✔ Successfully added labels autorelease: pending to issue: 18
✔ Successfully opened pull request available at url: https://api.github.com/repos/alojzy231/release-please-playground/pulls/20.
✔ Successfully opened pull request: 20.
✔ Successfully added labels autorelease: pending to issue: 20
```

**Comment by shinebayar-g** [comment]
Looks like it's same as https://github.com/googleapis/release-please/issues/1858

**Comment by alojzy231** [comment]
@shinebayar-g Thanks for catching this! This is probably the same problem but with different preconditions (steps to reproduce). I'll close this issue and look closely for answer in the issue you've mentioned
